client/components/pages/configuration/users/CreateUserDialog.vue

```diff
@@ -1,7 +1,8 @@
 <script lang="ts" setup>
-import { boolean, object, string, ref as yref } from 'yup'
+import { boolean, number, object, string, ref as yref } from 'yup'
 import FormDropdown from '~/components/input/FormDropdown.vue'
 import { CreateUserForm } from '~/utils/forms'
```

Comment:
honestly, sus capitalization

Reply:
I don't get it

Reply:
CreateUserForm is not a component, nor a class, it's just a global constant, so the PascalCase is weird, either use snakeCase or SCREAMING_CASE